srcpkgs/ghcup/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'ghcup'
+pkgname=ghcup
+version=0.1.17.4
+revision=1
+wrksrc="ghcup-hs-v$version"
+build_style='haskell-stack'
+make_build_args='--flag ghcup:tui --flag ghcup:internal-downloader'
```

Comment:
The internal downloader can cause issues with esoteric proxy configurations. In that case users can do

```sh
ghcup config set downloader Curl
```

to set the downloader back to curl, which can handle it. This should be a post-install message as well.

Reply:
I've added such a post-install message in `INSTALL.msg` now.